packages/perseus-editor/src/editor-page.tsx

```diff
@@ -83,15 +86,21 @@ class EditorPage extends React.Component<Props, State> {
     constructor(props: Props) {
         super(props);
 
+        // Convert any widgets that need to be converted to newer widget types
+        let convertedQuestionJson: PerseusRenderer = props.question;
+        if (props.question) {
+            convertedQuestionJson = convertDeprecatedWidgets(props.question);
+        }
+
+        const json = {
+            ..._.pick(this.props, "answerArea", "hints", "itemDataVersion"),
+            question: convertedQuestionJson,
+        };
+
         this.state = {
             // @ts-expect-error - TS2322 - Type 'Pick<Readonly<Props> & Readonly<{ children?: ReactNode; }>, "hints" | "question" | "answerArea" | "itemDataVersion">' is not assignable to type 'PerseusJson'.
-            json: _.pick(
-                this.props,
-                "question",
-                "answerArea",
-                "hints",
-                "itemDataVersion",
-            ),
+            json: json,
+            question: json.question,
```

Comment:
I'm not seeing where `this.state.question` is used elsewhere. Can you point me to where it is used? I'm also wonder if it needs to be updated in `changeJSON`.

Reply:
Whoops, thank you for catching this! It's a vestige of an old approach to fixing the json. Removed!